src/Seabug/CallContract.purs

```diff
@@ -182,19 +225,29 @@ buildContractConfig cfg = do
         , secure: cfg.serverSecureConn
         }
     , networkId: networkId
-    , slotConfig: defaultSlotConfig
-    , logLevel: cfg.logLevel
+    , logLevel: logLevel
     , extraConfig: { projectId: cfg.projectId }
     , wallet
     }
 
-buildNftList :: ListNftResult -> ListNftResultOut
+stringToLogLevel :: String -> Maybe LogLevel
+stringToLogLevel "Trace" = Just Trace
+stringToLogLevel "Debug" = Just Debug
+stringToLogLevel "Info" = Just Info
+stringToLogLevel "Warn" = Just Warn
+stringToLogLevel "Error" = Just Error
+stringToLogLevel _ = Nothing
+
+buildNftList :: NetworkId -> NftResult -> ListNftResultOut
 buildNftList
+  network
   { input: TransactionInput input, output: TransactionOutput output, metadata } =
   let
     transactionId = byteArrayToHex $ unwrap input.transactionId
     inputIndex = UInt.toInt input.index
-    address = addressBech32 output.address
+    -- TODO: What do we do if this fails?
```

Comment:
Interesting issue - can only occur if the NftResult passed in is malformed, which is built from a real transaction.
We could return an error? but the only way this would break is if you intentionally broke it.

Reply:
We could return an empty string like we're doing with `dataHash`. This could be acceptable if the only plans for using this field are to show it in the frontend.

I also don't see anywhere this is actually used in the frontend (besides a mock in a test), so we could just remove it, unless there are plans to use it in the future.

Reply:
Lets leave as is for now, get a ticket up, I'll check with Neil